base/system/services/database.c

```diff
@@ -1,4 +1,4 @@
-/*
+﻿/*
```

Comment:
Alerta @Doug-Lyons ! Did you accidentally change here UTF8woBOM to UTF8wBOM or what was that change here?

Reply:
I essentially only use 'Tortoise GIT' and its tools for editing source files. It seems that about once in 50 times, it will add an extra BOM at the beginning of the file for some unknown reason. I only notice this after a commit on Github and this shows it. I have fixed it now. Thanks.